app/helpers/application_helper.rb

```diff
@@ -573,7 +573,11 @@ def replacement_event_notice(event_schedule)
   end
 
   def quantity_left_of(resource)
-    "#{resource.quantity - resource.used}/#{resource.quantity}"
+    if resource.quantity.nil?
+      '-/-'
```

Comment:
This can be a single line:
`return '-/-' unless resource.quantity.present?`

It is generally a good practice to exit a function in the beginning, unless the necessary variables/values are set.
In our case, if there is no quantity, we should not continue. 
In other cases we may need a couple of values to be set, and if they aren't we just return nil or false or another default value, and don't continue further on. That way, particularly in a more complex method, you don't have to add a ton of if statements just to check the presence of a value.